### DISCUSSION TITLE
`print` modifiers seem to be lost when print orientation changes

### DISCUSSION THREAD

**Discussion by declanclark** [tailwindlabs/tailwindcss#10175] [Help]
I'm trying to use the `print` modifier to hide certain parts of my UI when the user tries to print the page i.e. `print:hidden`.

The preview in the dialogue looks as expected when I first press print. However, if I switch the orientation to landscape, everything seems to become visible again. Switch back to portrait and it's all hidden again. This behaviour was observed in Chrome and Firefox.

`print:hidden` doesn't seem to work at all for me in Safari.

Any ideas?

**Comment by ProPhycient5** [comment]
@declanclark Can you please share any gif to visualize the effect ??
